commands/env_testing.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/MichaelMure/git-bug/cache"
+	"github.com/MichaelMure/git-bug/repository"
+	"github.com/MichaelMure/git-bug/util/interrupt"
+)
+
+type testEnv struct {
+	env *Env
+	cwd string
+	out *bytes.Buffer
+}
+
+func newTestEnv(t *testing.T) *testEnv {
+	t.Helper()
+
+	cwd := t.TempDir()
+
+	repo, err := repository.InitGoGitRepo(cwd, gitBugNamespace)
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		require.NoError(t, repo.Close())
+	})
+
+	buf := new(bytes.Buffer)
+
+	backend, err := cache.NewRepoCache(repo)
+	require.NoError(t, err)
+
+	testEnv := &testEnv{
+		env: &Env{
+			repo:    repo,
+			backend: backend,
+			out:     out{Writer: buf},
+			err:     out{Writer: buf},
+		},
+		cwd: cwd,
+		out: buf,
+	}
+
+	cleaner := func(env *Env) interrupt.CleanerFunc {
```

Comment:
It's not wrong, but as you know you have an open backend, you could just do `interrupt.RegisterCleaner(testEnv.backend.Close) `

Reply:
Looking at that a second time - is it even necessary to catch `SIGTERM` or `SIGINT` during tests?  And this might actually be wrong because the back-end isn't also closed during test clean-up.

Reply:
Hmmm, I think closing twice is not a problem but ... ideally the test framework execute those `t.Cleanup` even on `SIGTERM`, `SIGINT` of panic? If so, we could simplify a bunch of tests that use defer and that interrupt cleaner.